lib/constants.js

```diff
@@ -0,0 +1,27 @@
+const TEMPLATE_ERRORS_TYPES = {
+  DISABLED: 'DISABLED',
+  SYNTAX_ERROR: 'SYNTAX_ERROR',
+  UNKNOWN: 'UNKNOWN',
+  BAD_URL: 'BAD_URL',
+  EXCEPTION: 'EXCEPTION',
+  MISSING: 'MISSING',
+  OTHER: 'OTHER',
+};
+
+const VSCODE_SEVERITY = {
+  DISABLED: 'Information',
+  SYNTAX_ERROR: 'Error',
+  UNKNOWN: 'Error',
+  BAD_URL: 'Warning',
+  EXCEPTION: 'Error',
+  MISSING: 'Warning',
+  OTHER: 'Information',
+};
+
+const WORD_REGEX = /{%.*(.*).*%}/;
```

Comment:
What does this do exactly? Might be a good idea to add a comment to explain it so people who are bad at regex (like me 😅 ) can quickly understand.

Reply:
Good call, I will update this name and add a comment